Docs/blog/new next js starter + gatsby starter name change (#1257)
vtex/faststore: docs/blog/2022-04-29-basestore.md

---

> description: Due to the launch of a Next.js base store starter, the repository of the existing base store, which used Gatsby, has changed.
> tags: [basestore]
> ---
> 
> # Base store repository name changed

Reviewer: ```suggestion
description: Due to the launch of a Next.js base store starter, the repository of the existing Base Store, which used Gatsby, has changed.
tags: [basestore]
---

# Base Store repository name changed
```

> The repository of the base store [starter](https://www.faststore.dev/starters), which uses Gatsby, has changed. It is now [gatsby.store](https://github.com/vtex-sites/gatsby.store).

Reviewer: ```suggestion
The repository of the Base Store [starter](https://www.faststore.dev/starters), which uses Gatsby, has changed. It is now [gatsby.store](https://github.com/vtex-sites/gatsby.store).
```

## Why did we make these changes?

> This change is due to the [launch of a Next.js base store starter](https://faststore.dev/releases/2022/04/29/basestore). This new starter is identical to the existing one in every feature, except it uses Next.js instead of Gatsby. Because of this, there are now two base store starters, in the repositories [nextjs.store](https://github.com/vtex-sites/nextjs.store) and [gatsby.store](https://github.com/vtex-sites/gatsby.store).

Reviewer: ```suggestion
This change is due to the [launch of the Base Store (Next.js) starter](https://faststore.dev/releases/2022/04/29/basestore). Since the Base Store is now compatible with both Next.js and Gatsby, there are now two Base Store repositories: [nextjs.store](https://github.com/vtex-sites/nextjs.store) and [gatsby.store](https://github.com/vtex-sites/gatsby.store).
```

## What needs to be done?

> Whenever you wish to access the base store repository, make sure you have the right repository name or URL for the desired framework:

Reviewer: ```suggestion
Whenever you need to access the Base Store repository, make sure you have the correct repository name and URL for the framework you are using:
```
